public/game-master/styles/game-master.styl

```diff
@@ -29,12 +29,15 @@ user-list-column-max-width = 400px
 			flex-grow 1.5
 
 			.level-control-info
+				position relative
 				display flex
 				justify-content space-between
 				margin-bottom 20px
 
 				.countdown-circle
-					font-size 1.5em
+					position absolute
```

Comment:
Because timer is inside container with Puzzle title and button only. So it should go outside the edges. That's my initial fault in mark-up :(